src/dune_file.ml

```diff
@@ -1175,15 +1175,21 @@ end
 
 module Install_conf = struct
   type file =
-    { src : string
-    ; dst : string option
+    { src : String_with_vars.t
+    ; dst : String_with_vars.t option
     }
 
   let file =
     peek_exn >>= function
-    | Atom (_, A src) -> junk >>| fun () -> { src; dst = None }
-    | List (_, [Atom (_, A src); Atom (_, A "as"); Atom (_, A dst)]) ->
-      junk >>> return { src; dst = Some dst }
+    | Atom (_, A _) ->
```

Comment:
This should be: `Atom _ | Quoted_string _ | Template _`

Reply:
Updated, thanks!